worlds/doom_1993/__init__.py

```diff
@@ -0,0 +1,296 @@
+from __future__ import annotations
+
+import logging
+from typing import List, Dict, Any, Set
+
+from BaseClasses import LocationProgressType, Region, Entrance, Location, Item, Tutorial, ItemClassification, MultiWorld
+from worlds.AutoWorld import World, WebWorld
+
+from . import Items
+from . import Locations
+from . import Options
+from . import Rules
+from . import Events
+from . import Regions
+
+
+logger = logging.getLogger("DOOM 1993")
```

Comment:
This looks unused. Maybe should be used in place of the `print` statement I flagged